src/python/py_speech_generation.cpp

```diff
@@ -0,0 +1,200 @@
+using ov::genai::PerfMetrics;
+using ov::genai::SpeechGenerationConfig;
+using ov::genai::SpeechGenerationPerfMetrics;
+using ov::genai::Text2SpeechDecodedResults;
+using ov::genai::Text2SpeechPipeline;
+
+namespace pyutils = ov::genai::pybind::utils;
+
+namespace {
+
+auto speech_generation_config_docstring = R"(
+    SpeechGenerationConfig
+    
+    Speech-generation specific parameters:
+    :param minlenratio: minimum ratio of output length to input text length; prevents output that's too short.
+    :type minlenratio: float
+
+    :param maxlenratio: maximum ratio of output length to input text length; prevents excessively long outputs.
+    :type minlenratio: float
+
+    :param threshold: probability threshold for stopping decoding; when output probability exceeds above this, generation will stop.
+    :type threshold: float
+)";
+
+auto speech_generation_perf_metrics_docstring = R"(
+    Structure with raw performance metrics for each generation before any statistics are calculated.
+
+    :param num_generated_samples: Returns a number of generated samples in output
+    :type num_generated_samples: int
+)";
+
+auto text_to_speech_decoded_results = R"(
+    Structure that stores the result from the generate method, including a list of waveform tensors
+    sampled at 16 kHz, along with performance metrics
+
+    :param speeches: a list of waveform tensors sampled at 16 kHz
+    :type speeches: list
+
+    :param perf_metrics: performance metrics
+    :type perf_metrics: SpeechGenerationPerfMetrics
+)";
+
+auto text_to_speech_generate_docstring = R"(
+    Generates speeches based on input texts
+
+    :param text(s): input text(s) for which to generate speech
+    :type text(s): str or list[str]
+
+    :param speaker_embedding optional speaker embedding tensor representing the unique characteristics of a speaker's
+                             voice. If not provided for SpeechT5 TSS model, the 7306-th vector from the validation set of the
+                             `Matthijs/cmu-arctic-xvectors` dataset is used by default.
+    :type speaker_embedding: openvino.Tensor or None
+
+    :param properties: speech generation parameters specified as properties
+    :type properties: dict
+
+    :returns: raw audios of the input texts spoken in the specified speaker's voice, with a sample rate of 16 kHz
+    :rtype: Text2SpeechDecodedResults
+)";
+
+SpeechGenerationConfig update_speech_generation_config_from_kwargs(const SpeechGenerationConfig& config,
+                                                                   const py::kwargs& kwargs) {
+    if (kwargs.empty())
+        return config;
+
+    SpeechGenerationConfig res_config = config;
+    if (!kwargs.empty())
+        res_config.update_generation_config(pyutils::kwargs_to_any_map(kwargs));
+
+    return res_config;
+}
+
+}  // namespace
+
+void init_speech_generation_pipeline(py::module_& m) {
+    m.doc() = "Pybind11 binding for Text-to-speech Pipeline";
```

Comment:
please, remove this Line as it overrides docs for whole genAI module (see changes in .pyi file)